coffee_core/src/coffee.rs

```diff
@@ -548,6 +548,50 @@ impl PluginManager for CoffeeManager {
         }
         Ok(nurse_actions)
     }
+
+    async fn tip(
+        &mut self,
+        plugins: &[&str],
```

Comment:
Why is `plugins` a vector?
I think it's better to be able to tip only plugin at a time or else there will be confusion
for example the user might think he is paying `amount_msat` for all the plugins but he is paying this amount for each pluin

Reply:
A single donation can be split across plugins or dependencies, we are not tipping X plugin because we can use just a single one bolt12, the vector here is for the all people that we tip

Reply:
> the vector here is for the all people that we tip

but the tip function we expose in the cmd accepts only 1 plugin at a time